typescript-intro/typescript-intro-quiz-notes.md

```diff
@@ -0,0 +1,48 @@
+# typescript-intro-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- When was TypeScript introduced?
+  TypeScript is a programming language that was created by Microsoft in October 2012.
+
+- Who created TypeScript?
+  It was created by Microsoft.
+
+- What does the TypeScript `Compiler` do?
+  A compiler is a software program that translates code from high-level programming languages like Java and C++ into machine code that a computer can execute. This process includes checking the syntax and structure of the code.
```

Comment:
Right, and a compiler can also just convert one language to another. The TS compiler converts your TS code into JS code so that the browser can run it...